packages/mongo-menu/src/MongoMenu.spec.tsx

```diff
@@ -1,45 +1,50 @@
 import React from 'react';
 import 'jest-dom/extend-expect';
 import { render, fireEvent, cleanup } from '@testing-library/react';
-import MongoMenu, { Product } from '.';
+import MongoMenu from '.';
 
 afterAll(cleanup);
 
 describe('packages/MongoMenu', () => {
+  const user = { name: 'Leafy', email: 'leafy@mongodb.com' };
   const onLogout = jest.fn();
   const onProductChange = jest.fn();
 
-  const { getByText } = render(
+  const renderedComponent = render(
     <MongoMenu
-      user={{ name: 'Leafy', email: 'leafy@mongodb.com' }}
-      activeProduct={Product.Atlas}
+      user={user}
+      activeProduct={'atlas'}
       onLogout={onLogout}
       onProductChange={onProductChange}
     />,
   );
 
+  const { getByText } = renderedComponent;
   const trigger = getByText('Leafy');
   fireEvent.click(trigger);
 
-  test('Atlas menu item links to cloud.mongodb.com', () => {
-    const atlasItem = getByText('Atlas').parentElement;
-    expect((atlasItem as HTMLAnchorElement).href).toBe(
-      'https://cloud.mongodb.com/',
+  const assertHrefByText = (expectedText: string, expectedHref: string) => {
```

Comment:
👍 We might consider moving this to lib so that we can reuse, but can do this when we want to leverage the pattern again later.